src/test/initialize.ts

```diff
@@ -99,7 +99,7 @@ async function closeWindowsInternal() {
             super("Command 'workbench.action.closeAllEditors' timed out");
         }
     }
-    const closeWindowsImplementation = (timeout = 15_000) => {
+    const closeWindowsImplementation = (timeout = 2_000) => {
```

Comment:
Didn't you already make this change in another PR @rchiodo 
I left this issue as I saw a related change in another PR.

Reply:
I thought I did. Maybe I didn't submit it.